src/aws.ts

```diff
@@ -38,7 +41,7 @@ import {
   makeIntegration,
 } from "./integration";
 import { AnyTable, isTable, ITable } from "./table";
-import type { AnyAsyncFunction } from "./util";
+import { AnyAsyncFunction } from "./util";
```

Comment:
why did this need to change?

Reply:
when the proxy still used to live in this file I had to use more utils here ... changed it back now 👍